src/app/modules/shared/feature-toggles/filters/feature-filter-types.ts

```diff
@@ -0,0 +1,3 @@
+export enum FeatureFilterTypes {
+  TARGETING = 'Microsoft.Targeting'
```

Comment:
What other targeting values we might have?

Reply:
I would say TimeWindow, UserPercentage, LocationBased, TenantBased or any other custom filter we might have in mind.